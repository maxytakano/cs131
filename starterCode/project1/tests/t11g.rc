int x;
int y[10];
bool a1[10];
bool a2[10][4];
bool a3[10][4][5];
bool z[10][4][5][1];

function : void main()
{
  x = y[x][x];
  x = z[x][x][x][x];
  x = z[x][x][x][x][x];
}
