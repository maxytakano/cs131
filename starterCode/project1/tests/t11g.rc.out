Error, "/tmp/cs131_testrunner_2FTaux/file.rc", line 10: 
  Type of expression referenced by array subscript (int) is not of array or pointer type.
Error, "/tmp/cs131_testrunner_2FTaux/file.rc", line 11: 
  Value of type bool not assignable to variable of type int.
Error, "/tmp/cs131_testrunner_2FTaux/file.rc", line 12: 
  Type of expression referenced by array subscript (bool) is not of array or pointer type.
Compile: failure.
